coldfront/core/allocation/models.py

```diff
@@ -487,6 +490,15 @@ def clean(self):
             except ValueError:
                 raise ValidationError(
                     'Invalid Value "%s" for "%s". Date must be in format YYYY-MM-DD' % (self.value, self.allocation_attribute_type.name))
+        elif expected_value_type == "JSON":
+            try:
+                _ = json.loads(self.value)
+            except:
+                raise ValidationError(
+                    'Invalid Value "%s" for "%s". Value must be valid JSON.' % (self.value, self.allocation_attribute_type.name))
```

Comment:
@peaches7083 [MINOR]
If the JSON is large, it might be hard to review the error. Could you put the malformed JSON at the end of the error message?

Reply:
Done.